neuralop/models/padding.py

```diff
@@ -54,36 +54,31 @@ def pad(self, x):
             padding = [int(round(p*r)) for (p, r) in zip(self.domain_padding, resolution)]
             
             print(f'Padding inputs of {resolution=} with {padding=}, {self.padding_mode}')
-
+            
+            
             output_pad = padding
-
-#             for scale_factor in self.output_scale_factor:
-#                 if isinstance(scale_factor, (float, int)):
-#                     scale_factor = [scale_factor]*len(resolution)
             output_pad = [int(round(i*j)) for (i,j) in zip(self.output_scale_factor,output_pad)]
 
             if self.padding_mode == 'symmetric':
                 # Pad both sides
-                unpad_indices = (Ellipsis, ) + tuple([slice(p, -p, None) for p in output_pad ])
+                unpad_indices = (Ellipsis, ) + tuple([slice(p, -p, None) for p in output_pad[::-1] ])
```

Comment:
Why did you reverse here?

Reply:
The first element of the output_pad pads around the last dimension. So, the first element should in the last slice.

Reply:
Why is that the case? Shouldn't it be in the same order?

Reply:
As per my understanding from the documentation, it is (left, right, top, botton, front, back) for 3D tensor... Left/right padding pads along the dim = -1. Top/bottom pads along the dim = -2... and so on.

Reply:
Great point @ashiq24 - can we make a quick comment and point to the doc for this?

Reply:
You meant in the code as a doc string?

Reply:
Yes, in the code just with `# comment` :)

Reply:
@ashiq24  can you add a short comment here?